service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftCMOControllerTest.java

```diff
@@ -179,6 +178,7 @@ void aboutToSubmitShouldPopulateCaseManagementOrder() throws Exception {
         assertThat(caseManagementOrder.getStatus()).isEqualTo(SELF_REVIEW);
     }
 
+    //TODO: caseDetails before is in this test as a start for conditional call to submitted code.
```

Comment:
Don't get this TODO. What's action required? is it TODO or just comment? Can we look at it and maybe fix / make it more clearer what's the problem?

Reply:
Again, inherited by FPLA-24. @lewisbirks can you explain todo?

Reply:
This looks like something sam added when he was trying to implement a conditional call to the progression controller, but seeing as it has been removed the comment and the case details before section can probably be removed.

Reply:
yeah unsure, why this comment would show as an addition 🤷‍♂